testsuite/do_tests.sh

```diff
@@ -127,6 +128,8 @@ TEST_OUTFILE="${REPORTDIR}/output.log"
 TEST_RETFILE="${REPORTDIR}/output.ret"
 TEST_RUNFILE="${REPORTDIR}/runtest.sh"
 
+TIME_LIMIT=120  # seconds, for Python tests
```

Comment:
Please clarify in the comment if this timeout is per test, or for all tests.
```suggestion
TIME_LIMIT=120  # seconds, for each of the Python tests
```